src/FOHInterface/FOHController.java

```diff
@@ -36,19 +37,26 @@ public int getAmountBooked(long timestamp) {
 
     @Override
     public void setServiceCharge(int newCharge) {
+
         Bill.setServiceChargePercentage(newCharge);
+        String notification = "The service charge now is: " + newCharge +"%";
+        Notification.addNotification(notification);
     }
 
     @Override
     public void markOrderComplete(int orderID) {
         OrderCollection.get(orderID).setComplete(true);
         OrderDAO.setOrderStatusToReady(orderID);
+        String notification = "The Order: " +  OrderCollection.get(orderID).getOrderId() + " is ready to collect.";
```

Comment:
If you've already got the orderID, why are you getting the orderId again?

Reply:
brains busted ill fix that